web/src/main/java/org/cbioportal/web/StudyController.java

```diff
@@ -68,6 +74,23 @@
         @RequestParam(required = false) StudySortBy sortBy,
         @ApiParam("Direction of the sort")
         @RequestParam(defaultValue = "ASC") Direction direction) {
+        
+        if (keyword == null &&
+            projection == Projection.SUMMARY &&
+            pageSize == 10000000 &&
+            pageNumber == 0 &&
+            sortBy == null &&
+            direction == Direction.ASC
+        ) {
```

Comment:
I imagine this paren is here to make it easier for adding/removing conditions?

Reply:
Oh sorry. I put trailing parens on a new line in personal projects and sometimes the reflex leaks into other work. I will revert this to the style seen in the rest of the codebase. Not trying to buck any trends.